### ISSUE TITLE
Link to Eva's page is private

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/neuro.polymtl.ca#37]
The team page:
![image](https://user-images.githubusercontent.com/2482071/142661254-2dd71547-de98-4ea6-9f97-c4809ab7709b.png)

Includes a link:
https://www.evaalonsoortiz.com/

Which brings there:
![image](https://user-images.githubusercontent.com/2482071/142661320-0deb2e78-9de2-479e-8942-7f7f0ded24ef.png)

I suggest redirecting to an open page for visibility-- potential applicants

**Comment by kousu** [comment]
Huh.

```
$ dig www.evaalonsoortiz.com
ext-sq.squarespace.com.
198.49.23.145
198.185.159.144
198.49.23.144
198.185.159.145
```

@evaalonsoortiz did your SquareSpace account perhaps expire?

**Comment by evaalonsoortiz** [comment]
I decided not to renew my membership and start using the Neuropoly website instead. I will fill out my details.

**Comment by jcohenadad** [comment]
Fixed in #40
